deploy/cr/cache/cache_1.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: infinispan.org/v2alpha1
+kind: Cache
+metadata:
+  name: example-cache
+spec:
+  adminAuth:
+    username:
+      key: username
+      name: basic-auth
+    password:
+      key: password
+      name: basic-auth
+  clusterName: example-infinispan
```

Comment:
do we need `clusterName`?

maybe this is somewhat future looking as the operator only controls one cluster at a time. in the case of controlling multiple clusters across different namespaces, then this could be useful. but, in that case, will it be mandatory that cluster names are unique? clusters in different namespaces cannot have the same name or the operator throws an error?

I just think this is an extra field that is not really needed. less fields, better. if the goal is to make sure that the cache gets created in the right cluster then maybe the k8s namespace is what should be used. make sense?

Reply:
Operator is (should be) able to handle several clusters in one namespace, that's why the cluster name is mandatory.
In the future we would probably add some more intelligence to the operator so it will be able to pick up the right cluster for the user cache.

Reply:
thanks! several clusters in one namespace makes sense.